Sources/SwiftStylable/Classes/Style/Dimensions/DimensionHolder.swift

```diff
@@ -8,7 +8,27 @@
 import UIKit
 
 open class DimensionHolder {
-    var size: CGFloat
+    
+    var reference: DimensionHolder?
+    
+    var _size: CGFloat?
+    var size: CGFloat {
+        set(newVal) {
```

Comment:
Setters hebben al een keyword 'newValue'. Als je geen bijzondere betekenis geeft in je eigen naam zou ik '(newVal) hier weg laten en gewoon 'newValue' gebruiken.

Reply:
Okay, ik zal hem weghalen. Ik vind de standaard newValue altijd een beetje magic dus maak altijd zelf nog een variable aan. Maar kan er wel mee leven ;)